web-ui/view/build.ml

```diff
@@ -35,13 +36,13 @@ let title_card ~status ~card_title ~hash_link ~ref_links ~first_created_at
                     h1
                       ~a:[ a_class [ "text-xl" ] ]
                       [ txt (Common.truncate ~len:80 card_title) ];
-                    (* a
-                       ~a:
-                         [
-                           a_class [ "btn btn-secondary btn-sm ml-4" ];
-                           a_href "#";
-                         ]
-                       [ txt "View Build History" ]; *)
+                    a
```

Comment:
Comment refers to line 24 above:

Change `~a:[ a_class [ "justify-between items-center flex" ] ]` to `~a:[ a_class [ "justify-between items-center flex space-x-4" ] ]`; currently there is no enforced spacing between the build history button and the 'ran-for' text. This matters only for very long commit messages, as in the screenshot below:

<img width="450" alt="Screenshot showing build history button next to the 'ran-for' text with no spacing" src="https://user-images.githubusercontent.com/13054139/206150441-66ac06e5-6b3d-465f-9612-b30f7366f67a.png">